symfony/3-nginx/etc/confd/templates/nginx/nginx_site.conf.tmpl

```diff
@@ -0,0 +1,27 @@
+server {
+    listen 80;
+
+    root /app/web;
+
+    server_name localhost;
+
+    location / {
+        try_files $uri /app.php$is_args$args;
+    }
+
+    location ~ ^/(app|app_dev|config)\.php(/|$) {
```

Comment:
once #60 is merged, only this location will be needed to be replaced. I'd be interested in considering how also minimising the replacement of this section in a good way